source/taskmanager/src/test/java/nl/aerius/taskmanager/WorkerPoolTest.java

```diff
@@ -64,9 +64,10 @@ public void messageDelivered(final MessageMetaData message) {
   }
 
   @Test
-  public void testWorkerPoolSizing() throws InterruptedException, IOException {
+  void testWorkerPoolSizing() throws InterruptedException, IOException {
     assertSame(0, workerPool.getCurrentWorkerSize(), "Check if workerPool size is empty at start");
-    workerPool.onQueueUpdate("", 10, 0, 0);
+    workerPool.onNumberOfWorkersUpdate(10, 0);
+    ;
```

Comment:
Missed it at first glance, but with the spotless changes it's a bit more obvious... Occurs a couple of times in this file.